subsys/net/lib/shell/net_shell_private.h

```diff
@@ -8,20 +8,40 @@
 #include <zephyr/shell/shell.h>
 #include <zephyr/net/net_ip.h>
 
-#define PR(fmt, ...)						\
-	shell_fprintf(sh, SHELL_NORMAL, fmt, ##__VA_ARGS__)
+#define PR(fmt, ...)                                                                               \
```

Comment:
Why do we have such a long lines, these are now max 100 chars long. Please adjust the length of the lines to be much smaller.